orchagent/fdborch.cpp

```diff
@@ -467,6 +581,12 @@ bool FdbOrch::addFdbEntry(const FdbEntry& entry, const string& port_name, const
         return true;
     }
 
+    /* Update entry's port name if needed */
+    if (entry.port_name.empty())
+    {
+        entry.port_name = port_name;
```

Comment:
In which case, the entry is created without port name? because

Reply:
We add an entry to m_entries in two places. One is storeFdbEntryState() and other is addFdbEntry(), Could you please make sure in both places we set port_name in entry before adding to m_entries?